pages/docs/modify/advanced/buses.js

```diff
@@ -0,0 +1,205 @@
+    const busNameNatural = abbreviate(busName)
+        .replace(/-[a-zA-Z]/g, g => ` ${g[1].toUpperCase()}`)
+        .replace(/^./, g => g.toUpperCase());
+    return <>
+        <h2 id={busName}>{busNameNatural}</h2>
+        <p>
+            <em>Package: <a href={`${treeRepoFiles}packages/bus-${busName}`}><code>@comunica/bus-{busName}</code></a></em>
+        </p>
+        <p>
+            {description}
+        </p>
+        <table>
+            <thead>
+            <tr>
+                <th>Actor</th>
+                <th>Package</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+            {actorsInfo}
+            </tbody>
+        </table>
+    </>
+}
+
+export default function Buses({ bussesInfo }) {
+  const busInfo = bussesInfo.map(busInfo => functionBusInfo(busInfo));
+  return (
+    <Template key={'/docs/modify/advanced/busbis/'}>
+    <div className="container-page">
+      <Head
+        title={'Buses and Actors'}
+        description={'An overview of all buses in Comunica and their actors.'}
+      />
+        <main>
+            <ul className="breadcrumbs">
+                <li><a href="/docs/">Documentation</a></li>
+                <li><a href="/docs/modify/">Modify Comunica</a></li>
+                <li><a href="/docs/modify/advanced/">Advanced modification</a></li>
+                <li>Buses and Actors</li>
+            </ul>
+            <div className="headers-overview">
+                <p>On this page</p>
+                <ol className="headers-overview-elements"/>
+            </div>
+
+            <h1>Buses and Actors</h1>
+            <hr/>
+            <p>
+                This page gives
+                an <strong>overview of all <em>buses</em> and <em>actors</em></strong> that
+                are used in the default Comunica engines, such
+                as <a href="https://github.com/comunica/comunica/tree/master/engines/query-sparql">Comunica
+                SPARQL</a> and <a href="https://github.com/comunica/comunica/tree/master/engines/query-sparql-file">Comunica
+                SPARQL File</a>.
+                Other configurations such as <a href="https://github.com/comunica/comunica-query-sparql-hdt">Comunica
+                SPARQL HDT</a> contain
+                additional actors and buses.
+            </p>
+            <p>
+                This builds upon
+                the <a href="/docs/modify/advanced/architecture_core/">core
+                architecture</a> of <em>actors</em>, <em>mediators</em>, and <em>buses</em>.
+                An overview of how these buses and actors are connected can be found in
+                the <a href="/docs/modify/advanced/architecture_sparql/">SPARQL architecture</a>.
+            </p>
+            {busInfo}
+        </main>
+    </div>
+    </Template>
+  )
+}
+
+async function fetchFile(url) {
+    return await (await fetch(url)).text();
+}
+
+async function fetchJson(url) {
```

Comment:
The Fetch API has a JSON method :-) https://developer.mozilla.org/en-US/docs/Web/API/Response/json